### PR TITLE
Address issues with color references

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

## Summary:

- remove legacy UI color reference and replace with new Atlas value
- deprecate colors for new Gray options
- deprecate unnecessary color values for new state colors

## Type of change:

Please delete options that are not relevant.

- [ ] New capability 
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: src/color/base.json [hunk 1]
```diff
@@ -4,70 +4,77 @@
...
+          "value": "dbdbdb",
+          "public": true
         },
         "500": {
-          "value": "00805d"
+          "value": "222222",
+          "public": true
         }
       },
       "manatee": {
```
Comment (blackfalcon) [inline src/color/base.json]: Needs a new category name

File: src/color/base.json [hunk 2]
```diff
@@ -4,70 +4,77 @@
...
         "comment": "{comments.color.base.value.comment}",
         "legacy": true
       },
-      "orca": {
-        "value": "222222",
-        "comment": "{comments.color.base.value.comment}",
-        "legacy": true
-      },
-      "sunset": {
+      "unionblue": {
```
Comment (blackfalcon) [inline src/color/base.json]: Needs a new category name

File: src/color/border.json [hunk 3]
```diff
@@ -51,33 +51,39 @@
...
+          "value": "{color.base.gray.200.value}",
           "public": true,
           "onDark": true,
           "usage": "Disabled border color on dark background",
           "wcag": "n/a"
         }
       },
       "Focus": {
-        "onLight": {
+        "newState": {
```
Comment (blackfalcon) [inline src/color/border.json]: This is a whole new action-based color category ... need a new name/reason

File: src/color/border.json [hunk 4]
```diff
@@ -51,33 +51,39 @@
...
           "value": "{color.base.black-opacity-15.value}",
           "public": true,
+          "usage": "Focus border color for neutral backgrounds",
```
Comment (blackfalcon) [inline src/color/border.json]: what is a 'neutral' background?

File: src/color/base.json (hunk 1 above)
Comment (blackfalcon) [reply src/color/base.json]: One of the blue/gray colors that don't really have a home. Current use:

* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/icon.json#L13
* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/text.json#L29

File: src/color/base.json (hunk 2 above)
Comment (blackfalcon) [reply src/color/base.json]: One of the blue/gray colors that don't really have a home. Current use:

* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/icon.json#L6
* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/text.json#L22

**Comment by cobrien64** [comment]
Sent Dale an updated file with the following changes:
- Fixed atlas hex color to correct hex value
- Added new category "neutral" to replace Manatee
- Added new grey colors but marked as non-public for now. UI review has confirmed these colors, but need to build the components that use them before making public.

File: src/color/border.json (hunk 3 above)
Comment (blackfalcon) [reply src/color/border.json]: @corobrie this one still needs your attention.

File: src/color/border.json (hunk 4 above)
Comment (blackfalcon) [reply src/color/border.json]: I am assuming that this is a reference to the new neutrals you added to base?

File: src/color/border.json (hunk 3 above)
Comment (corobrie) [reply src/color/border.json]: Need a bit more info on what this is trying to achieve (for my knowledge). Is this to denote a state of a component (active, focus, tapped, etc.)?

File: src/color/border.json (hunk 4 above)
Comment (corobrie) [reply src/color/border.json]: Yes!

File: src/color/ui.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
       },
       "disabled": {
         "onLight": {
-          "value": "{color.base.ui-opacity-20.value}",
+          "value": "{color.brand.atlas.400-opacity-20.value}",
```
Comment (corobrie) [inline src/color/ui.json]: Need to potentially change this based on if we make any changes to the disabled states.

File: src/color/brand.json [hunk 6]
```diff
@@ -69,13 +69,18 @@
...
         "400": {
-          "value": "0074ca",
+          "value": "0074cb",
           "comment": "{comments.color.brand.value.comment}",
           "public": true,
           "neutral": true,
           "usage": "Transactional colors to move guests through task-based interactions.",
           "wcag": "n/a"
         },
+        "400-opacity-20": {
```
Comment (corobrie) [inline src/color/brand.json]: What uses this token?

File: src/color/border.json (hunk 3 above)
Comment (blackfalcon) [reply src/color/border.json]: In your previous [pr](https://github.com/AlaskaAirlines/OrionDesignTokens/pull/22/files#diff-7c681f482810613828e7b41450aa7dfaR69) you suggested `gray`. 

This update here, within the context of `focus` still has `onDark` and `onLight`. I just used the placeholder name of `newState` with this note to ask; what do we want to name this?

File: src/color/ui.json (hunk 5 above)
Comment (blackfalcon) [reply src/color/ui.json]: ok, let me know what you want to do.

File: src/color/brand.json (hunk 6 above)
Comment (blackfalcon) [reply src/color/brand.json]: [color-ui-disabled-onLight](https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/ui.json#L52)

File: src/color/base.json [hunk 7]
```diff
@@ -35,17 +35,34 @@
           "public": true
         },
         "300": {
-          "value": "b2b2b2"
+          "value": "b2b2b2",
+          "public": false
```
Comment (blackfalcon) [inline src/color/base.json]: This flag is not needed. It only needs to be added when the value is to be `true`. `false` is implied.

File: src/color/base.json [hunk 8]
```diff
@@ -35,17 +35,34 @@
...
         },
         "400": {
-          "value": "767676"
+          "value": "767676",
+          "public": false
```
Comment (blackfalcon) [inline src/color/base.json]: See line #39

File: src/color/base.json [hunk 9]
```diff
@@ -35,17 +35,34 @@
...
         "500": {
           "value": "222222",
           "public": true
         }
       },
       "neutral": {
+        "100": {
+          "value": "f9fbfc",
+          "comment": "{comments.color.base.value.comment}",
+          "public": false
```
Comment (blackfalcon) [inline src/color/base.json]: See line #39

File: src/color/base.json [hunk 10]
```diff
@@ -35,17 +35,34 @@
...
+        },
+        "200": {
+          "value": "e2eaef",
+          "comment": "{comments.color.base.value.comment}",
+          "public": false
```
Comment (blackfalcon) [inline src/color/base.json]: See line #39

File: src/color/base.json [hunk 11]
```diff
@@ -35,17 +35,34 @@
...
+        },
+        "300": {
+          "value": "cbd8e2",
+          "comment": "{comments.color.base.value.comment}",
+          "public": false
```
Comment (blackfalcon) [inline src/color/base.json]: See line #39

**Pull request merged**